chains/corda/module/module.go

```diff
@@ -0,0 +1,27 @@
+package module
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/hyperledger-labs/yui-relayer/chains/corda"
+	"github.com/hyperledger-labs/yui-relayer/config"
+	"github.com/spf13/cobra"
+)
+
+type Module struct{}
+
+var _ config.ModuleI = (*Module)(nil)
+
+// Name returns the name of the module
+func (Module) Name() string {
+	return "corda"
+}
+
+// RegisterInterfaces register the ibc transfer module interfaces to protobuf Any.
+func (Module) RegisterInterfaces(registry codectypes.InterfaceRegistry) {
+	corda.RegisterInterfaces(registry)
```

Comment:
Now can we unexport `corda.RegisterIntefaces` now?
And can we make same changes in `fabric` and `tendermint`?

Reply:
To make the method private and move module.go under corda/, probably we need to refactor the package structure. This work is not done in this PR because it covers a rather large scope.

Reply:
Got it.